script/redirect-cc.py

```diff
@@ -4,20 +4,28 @@
 import os.path
 import os
 
+IS_WIN32 = sys.platform == 'win32'
+
 
 def main():
     args = sys.argv[1:]
-    replace_cc_arg(args)
+    brave_path = replace_cc_arg(args)
     if 'CC_WRAPPER' in os.environ:
         args = [os.environ['CC_WRAPPER']] + args
     cc_retcode = subprocess.call(args)
+    # To check the redirected file timestamp, it should be marked as dependency for ninja.
+    # Linux/MacOS gcc deps format includes this file properly.
+    # Windows msvc deps format does not include it, so we do it manually here.
+    if IS_WIN32 and cc_retcode == 0 and brave_path:
+        # This is a specially crafted string that ninja will look for to create deps.
+        sys.stderr.write('Note: including file: %s\n' % brave_path)
```

Comment:
```/showIncludes``` writes to stderr

https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/hdkef6tk(v=vs.90)?redirectedfrom=MSDN